flintrock/config.yaml.template

```diff
@@ -1,6 +1,8 @@
 services:
   spark:
     version: 1.6.0
+    # distribution: # optional; default to '2.6'
```

Comment:
Hmm, can we leave out the ability to specify distribution for now? I'm not sure about how best to name this option (e.g. there are non-Hadoop distributions like CDH, but we are assuming Hadoop) and, more importantly, I haven't fully considered the implications of supporting user-specified distributions.